### PR TITLE
[Improment]Add workload group query detail

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes
Add workload group query detail to Fe metric.

image

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/metric/PrometheusMetricVisitor.java [hunk 1]
```diff
@@ -46,6 +47,8 @@
  * doris_fe_job{job="load", type="mini", state="pending"} 0
  */
 public class PrometheusMetricVisitor extends MetricVisitor {
+
+    private static final Logger logger = Logger.getLogger(PrometheusMetricVisitor.class);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/metric/PrometheusMetricVisitor.java:51]: Wrong logger class

**Comment by wangbo** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]

**Pull request merged**